test/uk/gov/hmrc/apiplatformoutboundsoap/services/OutboundServiceSpec.scala

```diff
@@ -226,13 +223,34 @@ class OutboundServiceSpec extends AnyWordSpec with Matchers with GuiceOneAppPerS
         messageCaptor.getValue.messageId shouldBe messageId
         messageCaptor.getValue.globalId shouldBe expectedGlobalId
         messageCaptor.getValue.createDateTime shouldBe expectedCreateDateTime
+        messageCaptor.getValue.asInstanceOf[SentOutboundSoapMessage].ccnHttpStatus shouldBe httpCode
```

Comment:
Is the cast necessary? If you capture an OutboundSoapMessage it has this attribute accessible, I think.